src/aroma/aroma.c

```diff
@@ -158,8 +158,11 @@ byte libaroma_start() {
   
 #ifdef LIBAROMA_TTY_KDSETMODE
   ALOGI("KDSETMODE = KD_GRAPHICS");
-  int tty1 = open("/dev/tty1", O_RDWR);
+  char ttydev[32];
```

Comment:
that length seems arbitrary
- `/dev/tty1` == 9
- max value of `%i` == is platform specific.
- 0-terminator

I suggest doing this so you know the length is always 1
```c
uint8_t value = LIBAROMA_TTY_KDSETMODE ? 1 : 0;
`

With that, you can use a length of 11.